activestorage/app/models/active_storage/preview.rb

```diff
@@ -33,6 +33,8 @@
 class ActiveStorage::Preview
   class UnprocessedError < StandardError; end
 
+  delegate :forced_disposition_for_serving, :content_type_for_serving, :filename, to: :presentation
```

Comment:
What are your thoughts on including `ActiveStorage::Blob::Servable` here as well?  We would still need to delegate `filename` and `content_type`, but if new (required) methods were added to `ActiveStorage::Blob::Servable` in the future, we would automatically pick those up.

Reply:
Yes I agree!